doc-sdk-javascript_versioned_docs/version-1.x/core/initialization.mdx

```diff
@@ -24,7 +24,7 @@ let db: Surreal | undefined;
 
```

Comment:
`db` is already declared in line 23, by using const it is being redeclared in the function scope causing 
`used before being assigned` error in line 26